### PR TITLE
Add vtx smearing parameters: nominal beam parameters, realistic centroid

### PR DESCRIPTION
**Pull request by rmanzoni**
back port of https://github.com/cms-sw/cmssw/pull/10048

### CODE REVIEW

**Comment by civanch** [comment]
please test

**Comment by rmanzoni** [comment]
adding @mmusich and @diguida to the loop

**Comment by civanch** [comment]
+1

File: Configuration/StandardSequences/python/VtxSmeared.py [hunk 1]
```diff
@@ -38,7 +38,9 @@
     'Shifted15mmCollision2015'  :    'IOMC.EventVertexGenerators.VtxSmearedShifted15mmCollision2015_cfi',
     'NominalCollision2015'  :        'IOMC.EventVertexGenerators.VtxSmearedNominalCollision2015_cfi',
     'NominalHICollision2015':      'IOMC.EventVertexGenerators.VtxSmearedNominalHICollision2015_cfi',
-    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi'
+    'ZeroTeslaRun247324Collision'  : 'IOMC.EventVertexGenerators.VtxSmearedZeroTeslaRun247324Collision_cfi',
+    'RealisticCentroidNominalEmittance13TeVCollision': 'IOMC.EventVertexGenerators.RealisticCentroidNominalEmittance13TeVCollision_cfi',
```
Comment (franzoni) [inline Configuration/StandardSequences/python/VtxSmeared.py]: Hello Riccardo @rmanzoni ,
you should fix the path name IOMC.EventVertexGenerators.RealisticCentroidNominalEmittance13TeVCollision_cfi
to be consistent with the file you've added in IOMC
The same comment applies also to #10050 , #10049  , #10048  ,

**Comment by davidlange6** [comment]
@cmsbuild, please test 
(and notice the new commit)

**Comment by franzoni** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**